build-support/bin/protoc-gen-go/mac/10.13/1.1.0/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+FILENAME=${BASH_SOURCE[0]}
+DIR=$( cd "$( dirname "${FILENAME}" )" && pwd )
+export GOPATH=$DIR
+go get -u github.com/golang/protobuf/protoc-gen-go
```

Comment:
I think that this is assuming that the consumer has `golang` installed?

Reply:
That was intentional, since we don't have a dedicated macOS environment, so I thought using whatever is on the machine would be best. Since downloading Go is simply unpacking a tarball, I'll update this to fetch a specific version. Just a sec...

Reply:
Well, hold on... I'm commenting on the other review currently.